pkg/scheduler/scheduler.go

```diff
@@ -38,6 +38,8 @@ import (
 	"google.golang.org/grpc/status"
 
 	"github.com/grafana/mimir/pkg/frontend/v2/frontendv2pb"
+	"github.com/grafana/mimir/pkg/mimirpb"
+	"github.com/grafana/mimir/pkg/querier/querierpb" //lint:ignore faillint we can't avoid using this given that's where the Protobuf definition lives
```

Comment:
Should we remove the lint rule? What was it trying to protect against?
It doesn’t seem very satisfactory to say effectively “I need this”.

Reply:
Given the approval, I'm going to merge this - happy to address this in a follow up PR.